.github/workflows/basic.yml

```diff
@@ -208,11 +210,28 @@ jobs:
     - name: Checkout
       uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
+    - name: Get CMake for lukka/run-vcpkg
+      uses: lukka/get-cmake@latest
```

Comment:
please use hash commits in all actions, for specific versions (pls see e.g. checkout action, above)

`@latest` is definetely a no-go 😉

Reply:
Done, but not pushed out.

Reply:
😄 the one under which is my issue

We shouldn't use `lukka/run-vcpkg@v11.5` like this - we have to use a hash sha corresponding to the tag, and tag put only in the comment (for us, for readability)

example of properly used action is above - the `checkout` action.
<br />

Solution for you (please apply everywhere):
```
lukka/run-vcpkg@5e0cab206a5ea620130caf672fce3e4a6b5666a1 # v11.5
```

Reply:
Done

Reply:
> This issue is under:
> ```
>     - name: Get CMake for lukka/run-vcpkg
>       uses: lukka/get-cmake@latest
> ```

haha, ok, mea culpa I didn't realize it's the old action 😄 

thx!